app/model/document.py

```diff
@@ -56,7 +56,7 @@ class DocumentCreateDTO(BaseModel):
     # From PhysicalDocument
     title: str
     source_url: str
-    user_language_name: str
+    user_language_name: Optional[str]
```

Comment:
This fixes https://linear.app/climate-policy-radar/issue/PDCT-666/language-should-not-be-mandatory